### PR TITLE
panlint: ignore heredoc blocks

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: panc/src/main/scripts/panlint/panlint.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
 RE_COMMENT_LINE = re.compile(r'^\s*' + RS_COMMENT + '.*$')
 RE_ANNOTATION = re.compile(r'@\w*{.*?}', re.S)
 RE_OPERATOR = re.compile(r'([>=<!?]=|[+*=/-])')
+RE_HEREDOC = re.compile(r'<<(\w+);\s*$.*?\1$', re.S|re.M)
```
Comment (ned21) [inline panc/src/main/scripts/panlint/panlint.py]: pep8 prefers:
```
RE_HEREDOC = re.compile(r'<<(\w+);\s*$.*?\1$', re.S | re.M)
```

which seems to be borne out by https://docs.python.org/3/howto/regex.html#compilation-flags

File: panc/src/main/scripts/panlint/panlint.py [hunk 2]
```diff
@@ -266,8 +267,21 @@ def find_annotation_blocks(text):
         start_char, end_char = annotation.span()
         start_line = text[:start_char].count('\n') + 1
         end_line = start_line + text[start_char:end_char].count('\n')
-        for i in range(start_line, end_line + 1):
-            result.append(i)
+        result += range(start_line, end_line + 1)
+    return result
+
+def find_heredoc_blocks(text):
```
Comment (ned21) [inline panc/src/main/scripts/panlint/panlint.py]: Double newline for pep8

**Comment by stdweird** [comment]
@ned21 fixed (but i do not understand what yo umean with the "born out" remark)

**Comment by ned21** [comment]
@stdweird `re.S|re.M` seemed more logical to me but the python3 docs confirm that `re.S | re.M` is the preferred way of doing it.

**Pull request merged**